test/tests/server_connectorTest.js

```diff
@@ -14,10 +14,12 @@ describe("Connector Server", function () {
 		
 		win = yield loadZoteroPane();
 		connectorServerPath = 'http://127.0.0.1:' + Zotero.Prefs.get('httpServer.port');
-		testServerPath = 'http://127.0.0.1:' + testServerPort;
 	});
 	
 	beforeEach(function () {
+		// Alternate ports to prevent exceptions not catchable in JS
+		testServerPort += (testServerPort & 1) ? 1 : -1;
```

Comment:
I've successfully ran Server Connector tests every time after this change. Before that it was 50/50. And 50% success rate is also good argument for that the ports do get closed eventually - may be a matter of miliseconds. I have not investigated this further and just went for the easy solution that works. And if it doesn't - we can always investigate.